webapp/channels/src/actions/emoji_actions.js

```diff
@@ -132,6 +134,35 @@ export function loadCustomEmojisForCustomStatusesByUserIds(userIds) {
     };
 }
 
+export function loadCustomEmojisForRecentCustomStatuses() {
+    return (dispatch, getState) => {
+        const state = getState();
+        const customEmojiEnabled = isCustomEmojiEnabled(state);
+        const customStatusEnabled = isCustomStatusEnabled(state);
+        if (!customEmojiEnabled || !customStatusEnabled) {
```

Comment:
Since we're not reusing the above flags, could we simplify?
```suggestion
        if (!(isCustomEmojiEnabled(state) && isCustomStatusEnabled(state))) {
```

Reply:
This was copied from one of the selectors, so I'd lean towards not changing it. I could see combining it, but I'd lean towards `if (!isCustomEmojiEnabled(state) || isCustomStatusEnabled(state)) {` since that's the pattern we tend to use elsewhere.

I'm trying to think why that is since "!(A && B)` is how I'd think about this logic in my head, but we've written `!A || !B` so many times that the `&&` version looks weird